src/olympia/promoted/migrations/0030_remove_promotedaddon_addon_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.20 on 2025-05-01 19:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('discovery', '0016_delete_promotedaddon_discoveryaddon_and_more'),
+        ('addons', '0054_update_default_locale_es_to_es-es'),
+        ('promoted', '0029_auto_20250429_1315'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='promotedaddon',
+            name='addon',
+        ),
+        migrations.RemoveField(
+            model_name='promotedapproval',
+            name='version',
+        ),
+        migrations.DeleteModel(
+            name='PromotedTheme',
+        ),
+        migrations.DeleteModel(
+            name='PromotedAddon',
+        ),
+        migrations.DeleteModel(
+            name='PromotedApproval',
+        ),
```

Comment:
This is risky to do in the same deploy to prod as the code being removed - the underlying sql will drop the tables.  Remove it from the migration and add a follow-up to re-generate the migrations in a subsequent sprint.

(PromotedTheme is fine to delete, assuming the references to that model were dropped in a previous deployment)